protobuf/lib/src/protobuf/field_info.dart

```diff
@@ -6,7 +6,7 @@ part of protobuf;
 
 /// An object representing a protobuf message field.
 class FieldInfo<T> {
-  FrozenPbList<T>? _emptyList;
+  PbList<T>? _emptyList;
```

Comment:
Is this still always frozen?
Maybe add a comment about that?

Reply:
It is frozen, yes. Otherwise you could modify the "empty list" and calls to `readonlyDefault` would return a list with elements.

I'll document it.

Reply:
@sigurdm PTAL.